src/cdk/tree/tree.ts

```diff
@@ -1247,6 +1264,17 @@ export class CdkTreeNode<T, K = T> implements OnDestroy, OnInit, TreeKeyManagerI
     return this._tree._getPositionInSet(this._data);
   }
 
+  protected _getTabindex(): number | null {
+    if (!this._tree._keyManager) {
+      return null;
+    }
+    // TODO: determine if this equailty check will work in SSR
+    if (this._tree._keyManager.getActiveItem() === this) {
```

Comment:
TODO: see if this has any change detection issues.

TODO: consider putting select/deselect methods on TreeKeyManagerItem.